### ISSUE TITLE
Outpost Security NPCs only attack thieves if they can stun the target

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#12807]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Outpost Security NPCs only seem to attack thieves if they are equipped with a weapon that primarily does stun, like a stun baton or a stun gun. 

If equipped with any other weapon type, it won't do anything

This includes lethal items and ammo (shotgun with shells, axe, knives), but also non-lethal (shotgun with rubber shell, syringe gun with chloral hydrate).

If NPCs have no option to stun they should default to use ranged weapons of any type, then melee weapons against thieves

https://github.com/Regalis11/Barotrauma/blob/ca2c30edca3e0edf6ad69cf35ea62024793c4730/Barotrauma/BarotraumaShared/SharedSource/Characters/AI/Objectives/AIObjectiveCombat.cs#L658

### Reproduction steps

1. Start a campaign
2. Control security
3. Remove stun baton and stun gun
4. Steal something in front of security

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.0.21.0

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
Not being able to shoot any weapon breaks the AI if you create NPCs that don't specifically have Stun batons/guns. 
This means security NEEDS those items to function

Even giving security NPCs alternative "stun" methods, they won't use them (rubber shotgun shells, chloral hydrate in a syringe gun).

For modding this also means you can't make more "hostile" factions/outposts/npcs

**Comment by itchyOwl** [comment]
I don't think we should make the security use lethal weapons when they try to arrest, unless the target is aggressive. Also the bots shouldn't be broken, they just don't attack (just threaten). But I see that this might not work with all modded content.

I think at least those alternative stunning methods  should be taken into account (added after the behavior was implemented -> the bots just aren't aware of them). Would be good to make it possible to turn off the arresting too. And if it's possible for the players to disarm the security, that should count as an aggressive action that triggers offensive behavior.
